src/e2e/features/4-external-systems.feature

```diff
@@ -1,8 +1,16 @@
+# 
+# SPDX-FileCopyrightText: 2024 Lifely
+# SPDX-License-Identifier: EUPL-1.2+
+#
+
 Feature: External Systems
 
   Scenario: Employee wants to create a new document for a zaak
     Given "Bob" is logged in to zac
     And Employee "Bob" is on the newly created zaak with status "Intake"
+    
     When Employee "Bob" clicks on Create Document for zaak
-    And Employee "Bob" closes the SmartDocuments tab
-    Then Employee "Bob" should not get an error
+    And Employee "Bob" fills in the create document form
+    And Employee "Bob" submits the form to create the document should see SmartDocuments tab
```

Comment:
The bit "should see SmartDocuments tab" (I'm not clear what that is btw) looks like a `Then` clause to me 🤔

Reply:
Ah I get it now, that refers to a browser tab 👍 
So, then I should reword this to indicate it is what happens, like so:
```gherkin
And Employee "Bob" submits the form to start the SmartDocuments wizard
```
Underlying code is fine though 😄

Reply:
Totally agreeing here; but the `then` is the failing step that needs to be fixed in either our backend code, or the setup of the callback page or even other. There in no 'then' yet.

Reply:
And leaving it there calling an empty function seems to me not adding any value;  I removed that whole function btw

Reply:
Adjusted it; thxz!